File: app/src/main/java/com/neome/java/api/meta/base/dto/StudioFieldIcon.java
// neome.ai API - do not change
//

package com.neome.java.api.meta.base.dto;

import com.neome.java.api.meta.base.Types.MetaIdField;
import com.neome.java.api.meta.base.Types.MetaIdVar;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class StudioFieldIcon extends StudioFieldEditable
{
  @Nullable
  public MetaIdField defaultFieldId;

  @Nullable
  public String defaultValue;

  @Nullable
  public MetaIdVar defaultVarId;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioFieldIcon.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.MetaIdField
import com.neome.api.meta.base.Types.MetaIdVar

class StudioFieldIcon : StudioFieldEditable() {
    var defaultFieldId: MetaIdField? = null
    var defaultValue: string? = null
    var defaultVarId: MetaIdVar? = null
}
